src/Session.php

```diff
@@ -211,7 +214,7 @@ public function getId()
      */
     public function regenerateId()
     {
-        if (!$this->idRegenerated) {
+        if (! $this->idRegenerated) {
```

Comment:
Please try to be consistent and don't change existing coding style (unless it violates one of PSR-x standards?).

Reply:
PSR-2

Reply:
Well, you mean the (! ...

Ok I understand!